sdk/core/core-tracing/lib/utils/getTraceParent.ts

```diff
@@ -0,0 +1,21 @@
+import { SpanContext } from "../interfaces/span_context";
+import { TraceFlags } from '../interfaces/trace_flags';
+
+const VERSION = "00";
+
+/**
+ * Generates a `traceparent` value given a span context. 
+ * @param spanContext Contains context for a specific span.
+ * @returns The `spanContext` represented as a `traceparent` value.
+ */
+export function getTraceParent(spanContext: SpanContext): string {
+  if (!spanContext.traceId || !spanContext.spanId) {
+    throw new Error(`Missing required fields "traceId" or "spanId" from spanContext.`);
```

Comment:
Related to another comment, what do you think about returning null if these checks fail, and logging the errors instead of throwing? That fits better with the overall philosophy that turning on tracing shouldn't break user's applications.

Reply:
Love it